.github/workflows/_rust.yml

```diff
@@ -39,6 +39,7 @@ env:
 jobs:
   rust-lints:
     uses: ./.github/workflows/_rust_lints.yml
+    if: (!cancelled() && !failure())
```

Comment:
Is this because these are required to run?

Reply:
We discussed that it is not necessary but it also doesn't hurt.